src/AtomsGaffer/AtomsCrowdGenerator.cpp

```diff
@@ -718,7 +718,7 @@ ConstCompoundObjectPtr AtomsCrowdGenerator::computeBranchAttributes( const Scene
         }
 
         // Need to find with point has the data of the current agent
-        std::vector<int> agentIdVec = agentIdData->readable();
+        const std::vector<int> &agentIdVec = agentIdData->readable();
```

Comment:
John H: `AtomsCrowdGenerator::computeBranchAttributes()` appears to be inadvertently copying the agentId vector. Should be `const &` instead? 100000 copies of 100000 things is a lot of copying.